tests/db_tests/resources/db_seed_data.py

```diff
@@ -28,3 +31,193 @@
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    ],
+}
+
+PROCUREMENT_DATA = {
+    "id": [uid["uuid16"], uid["uuid17"], uid["uuid18"]],
+    "construction_contract": ["Contract_1", "Contract_2", "Contract_3"],
+    "project_id": [uid["uuid10"], uid["uuid11"], uid["uuid12"]],
+    "start_date": [datetime.date(2021, 4, 1), datetime.date(2022, 11, 3), datetime.date(2023, 1, 14)],
+    "end_date": [datetime.date(2022, 3, 20), datetime.date(2023, 3, 10), datetime.date(2024, 1, 13)],
+    "status": [enums.StatusEnum.COMPLETED, enums.StatusEnum.NOT_YET_STARTED, enums.StatusEnum.ONGOING_DELAYED],
+    "procurement_status": [
+        enums.ProcurementStatusEnum.AWARDING_OF_CONSTRUCTION_CONTRACT,
+        enums.ProcurementStatusEnum.EVALUATION_OF_TENDERS,
+        enums.ProcurementStatusEnum.PUBLICATION_OF_ITT,
+    ],
+    "comments": [
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        "sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    ],
+}
+
+PROJECT_PROGRESS_DATA = {
+    "id": [uid["uuid19"], uid["uuid20"], uid["uuid21"]],
+    "package_id": [uid["uuid7"], uid["uuid8"], uid["uuid9"]],
+    "answer_1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    "answer_2": "sed do eiusmod tempor incididunt ut labore et dolore magna",
+    "answer_3": "Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+    "answer_4": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    "answer_5": "sed do eiusmod tempor incididunt ut labore et dolore magna",
+    "answer_6": "Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+}
+
+DIRECT_FUND_DATA = {
+    "id": [uid["uuid22"], uid["uuid23"], uid["uuid24"]],
+    "project_id": [uid["uuid10"], uid["uuid11"], uid["uuid12"]],
+    "start_date": [datetime.date(2021, 4, 1), datetime.date(2022, 11, 3), datetime.date(2023, 1, 14)],
+    "end_date": [datetime.date(2022, 3, 20), datetime.date(2023, 3, 10), datetime.date(2024, 1, 13)],
+    "state": [enums.StateEnum.ACTUAL, enums.StateEnum.FORECAST, enums.StateEnum.FORECAST],
+    "amount": [1000000.00, 12345678.00, 0.00],
+    "pra_or_other": [enums.PRAEnum.OTHER, enums.PRAEnum.PRA, enums.PRAEnum.OTHER],
+    "contractually_committed_amount": [1000000.00, 12345678.00, 0.00],
+}
+
+CAPITAL_DATA = {
+    "id": [uid["uuid25"], uid["uuid26"], uid["uuid27"]],
+    "project_id": [uid["uuid10"], uid["uuid11"], uid["uuid12"]],
+    "start_date": [datetime.date(2021, 4, 1), datetime.date(2022, 11, 3), datetime.date(2023, 1, 14)],
+    "end_date": [datetime.date(2022, 3, 20), datetime.date(2023, 3, 10), datetime.date(2024, 1, 13)],
+    "state": [enums.StateEnum.ACTUAL, enums.StateEnum.FORECAST, enums.StateEnum.FORECAST],
+    "amount": [2, 000, 000.00, 0.00, 10, 345, 000.00],
```

Comment:
It doesn't make the code fall over so not major but I think we only need 3 values here? The same for the amount value in `INDIRECT_FUND_UNSECURED_DATA`

Reply:
Looks like `INDIRECT_FUND_UNSECURED_DATA` still has 5 values